test/runtests.jl

```diff
@@ -56,11 +57,22 @@ arrays = setup_backends()
                 all(σ[2:end-1, 1] .== σ[2:end-1, 2]) && all(σ[2:end-1, end] .== σ[2:end-1, end-1])
 
         @allowscalar u[end,:,1] .= 3
-        BC!(u, U, true) # save exit values
+        BC!(u,U,true) # save exit values
         @allowscalar @test all(u[end, :, 1] .== 3)
 
         WaterLily.exitBC!(u,u,U,0) # conservative exit check
         @allowscalar @test all(u[end,2:end-1, 1] .== U[1])
+
+        BC!(u,U,true,(2,)) # periodic in y and save exit values
```

Comment:
Not sure why, but the code cov tool doesn't seem to think these tests are actually running the new code. Give it a look.

Reply:
Yes, I saw that the codecov doesn't register a few lines